a.cpp

```diff
@@ -0,0 +1,67 @@
+#include <stdlib.h>
+#include <gflags/gflags.h>
+#include <iostream>
+
+#define INVALID_FLAG_ERROR "Invalid flag combinations"
+#define MISSING_ARGUMENT_ERROR "Missing arguments"
+
+DEFINE_string(registeruser, "", "Registers the given username");
+DEFINE_string(user, "", "Logs in as the given username");
+DEFINE_string(warble, "", "Creates a new warble with the given text");
+DEFINE_string(reply, "", "Indicates that the new warble is a reply to the given id");
+DEFINE_string(follow, "", "Starts following the given username");
+DEFINE_string(read, "", "Reads the warble thread starting at the given id");
+DEFINE_bool(profile, false, "Get the user's profile of following and followers");
+
+// Prints error message
+void PrintError(char* errorMessage, char* fields) {
```

Comment:
Better to change signature to take in const reference std::strings

Reply:
OK